calcom/cal.com
# feat: Enable drag and drop of event types

## What does this PR do?



Fixes #10016
/claim #10016




https://github.com/calcom/cal.com/assets/47350983/6a4ab7da-e4d7-47d8-b8f7-e5f058905f58


https://github.com/calcom/cal.com/assets/47350983/a4881ee8-362d-42bf-b687-fda2bfd855af





## Type of change



- New feature (non-breaking change which adds functionality)

## How should this be tested?



In the event-types page, move events around by dragging and dropping them. Click on refresh to ensure that the order is preserved. Works with mobile as well.

## Mandatory Tasks


## Checklist



- I haven't checked if my PR needs changes to the documentation
- I haven't added tests that prove my fix is effective or that my feature works
